scrapping/with_scrapy/data.json

```diff
@@ -1,50 +1,290 @@
 [
-{"URL": "https://www.bbc.com/news/live/world-europe-65851660", "Tag": "Europe", "Title": "Ukraine on attack in south to split Russian stronghold", "Image": "LIVE"},
```

Comment:
@Prasad-Bhogale typically we don't commit data into git. Please delete this file